src/Formatter.spec.ts

```diff
@@ -251,6 +251,24 @@ describe('Formatter', () => {
                 end sub
            `);
         });
+
+        it('formats outdents', () => {
```

Comment:
can you add another test to see what happens if we have a line with double-indent, but then a few lines later we don't have a double-outdent? 

```brighterscript
key_9: { env: ["any"], themes: ["any"], runtimeCheck: function() as boolean ' 2 indenters
        return true
    end function ' 1 outdenter
} ' 1 outdenter
```

Reply:
Here are some additional test blocks that are formatted correctly.

```brighterscript
function test()
    temp = {
        key_9: { env: ["any"], themes: ["any"], runtimeCheck: function() as boolean
                return true
        end function }
    }
end function

function test()
    temp = {
        key_9: { env: ["any"], themes: ["any"], runtimeCheck: function() as boolean
                return true
            end function
        }
    }
end function

function test()
    temp = {
        key_9: {
            env: ["any"], themes: ["any"], runtimeCheck: function() as boolean
                return true
            end function
        }
    }
end function
```

Reply:
updated